splink/linker.py

```diff
@@ -2080,69 +2080,36 @@ def cluster_pairwise_predictions_at_threshold(
         return cc
 
     def profile_columns(
-        self, column_expressions: str | list[str] = None, top_n=10, bottom_n=10
+        self,
+        column_expressions: str | list[str],
+        top_n=10,
+        bottom_n=10,
+        distribution_plots=True,
```

Comment:
You might as well add some type annotations

Reply:
Added